src/main/java/io/leonis/algieba/geometry/Rotation.java

```diff
@@ -0,0 +1,28 @@
+package io.leonis.algieba.geometry;
+
+import lombok.experimental.UtilityClass;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.factory.Nd4j;
+
+/**
+ * The Class Rotation.
+ *
+ * @author Rimon Oz
+ */
+@UtilityClass
+public class Rotation {
```

Comment:
I see you have servaral UltilityClasses and interfaces with default methods. I rather see you use one of the two. Preferably the interface.